src/System.Buffers.Primitives/System/Buffers/ReadOnlyBuffer_helpers.cs

```diff
@@ -0,0 +1,203 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Collections.Sequences;
+using System.Runtime.CompilerServices;
+
+namespace System.Buffers
+{
+    public readonly partial struct ReadOnlyBuffer 
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        static bool TryGetBuffer(Position begin, Position end, out ReadOnlyMemory<byte> data, out Position next)
```

Comment:
internal

Reply:
Why internal? It can be private

Reply:
The same thing, there is a test in tests/System.Buffers.Primitives.Tests/ReadableBufferFacts.cs  using it.